docs/en/observability/slo-troubleshoot.asciidoc

```diff
@@ -0,0 +1,258 @@
+[[slo-troubleshoot-slos]]
+= Troubleshoot service-level objectives (SLOs)
+
+++++
+<titleabbrev>Troubleshoot SLOs</titleabbrev>
+++++
+
+include::slo-overview.asciidoc[tag=slo-license]
+
+[WARNING]
+====
+Do not edit, delete, or tamper with any "internal" assets mentioned in this document, such as the transforms or ingest pipelines created by the SLO application.
+
+Do not attempt to edit the `.slo-observability.*` indices mentioned in this document by overriding index templates or editing the settings/mappings.
+
+The implementation details described here are subject to change.
+====
+
+This document provides an overview of common issues encountered when working with service-level objectives (SLOs). It explores the relationships between SLOs and other core functionalities within the stack, such as {ref}/transforms.html[transforms] and {ref}/ingest.html[ingest pipelines], highlighting how these integrations can impact the functionality of SLOs.
+
+* <<slo-understanding-slos>>
+* <<slo-common-problems>>
+* <<slo-troubleshoot-actions>>
+* <<slo-troubleshoot-beta>>
+
+[discrete]
+[[slo-understanding-slos]]
+== Understanding SLO internals
+
+[TIP]
+====
+If you’re already familiar with how SLOs work and their relationship with other system components, such as transforms and ingest pipelines, you can jump directly to <<slo-common-problems>>.
+====
+
+An SLO is represented by several system resources:
+
+* *SLO Definition*: Stored as a Kibana Saved Object.
+* *Transforms*: For each SLO, {kib} creates two transforms:
+** *Rolling-up transform*: `slo-{slo.id}-{slo.revision}`, rolls up the data into a smaller set of documents. The source indices of this transform are defined by the SLO. The target index will be `.slo-observability.sli-v{slo.internal-version}-{monthly date}`.
+** *Rolling-up ingest pipeline*: `slo-observability.sli.pipeline-{slo.id}-{slo.revision}`, used by the rolling-up transform.
+** *Summarizing transform*: `slo-summary-{slo.id}-{slo.revision}`, updates the latest values, such as the observed SLI or remaining error budget, for efficient searching and filtering of SLOs. The source of this transform is `.slo-observability.sli-v{slo.internal-version}*`. The target index is `.slo-observability.summary-v{slo.internal-version}`.
+** *Summarizing ingest pipeline*: `slo-observability.summary.pipeline-{slo.id}-{slo.revision}`, used by the summarizing transform.
+
+* *Additional resources*: {kib} also installs and manages shared resources to the SLOs, including index templates, indices, and ingest pipelines, among others.
+
+Each time an SLO is updated, the associated assets (transforms & ingest pipelines) are updated with a new `{slo.revision}`.
```

Comment:
Thanks a lot! I have updated it to:

> When an **SLO update** changes any of the `SLI parameters`, the `SLO objective`, or the `time window`, a revision bump (`{slo.revision}`) and a full reinstallation of the associated assets (transforms and ingest pipelines) occur. In addition, the revision bump deletes any previously aggregated data for that SLO. Updates to fields like `name`, `description`, or `tags` do not trigger a revision bump or asset reinstallation.

cc: @dedemorton , in case you want to review the sentence too... (already commited, feel free to review directly in the new code).